aws/resource_aws_s3_bucket_object_test.go

```diff
@@ -1233,8 +1234,8 @@ resource "aws_s3_bucket_object" "object" {
   bucket                 = aws_s3_bucket.object_bucket.bucket
   key                    = "test-key"
   server_side_encryption = "AES256"
-  source                 = %[1]q
-  etag                   = filemd5(%[1]q)
+  source                 = %[2]q
+  etag                   = filemd5(%[2]q)
```

Comment:
👍 It might be worthwhile to use `%[1]d` to `bucket` for consistency here and below

Reply:
I have changed all to be more consistent and updated the test output